packages/core/lib/commands/test/determineTestFilesToRun.js

```diff
@@ -1,15 +1,23 @@
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+const walkdir = require("walkdir");
+
 const determineTestFilesToRun = ({ inputFile, inputArgs = [], config }) => {
-  const path = require("path");
-  const fs = require("fs");
-  const glob = require("glob");
   let filesToRun = [];
-
   if (inputFile) {
     filesToRun.push(inputFile);
   } else if (inputArgs.length > 0) {
-    inputArgs.forEach(inputArg => filesToRun.push(inputArg));
+    for (let fileOrDir of inputArgs) {
```

Comment:
`walkdir.sync` will throw if any member of `inputArgs` does not exist. `inputArgs` should be filtered to only include valid folders, possibly with some messaging to alert the user of the invalid entries.

Reply:
Additionally, if a user specifies empty directories, the next if statement checking `filesToRun.length` will  pass and the default test directory suite will run. This is strange behavior if a user specifically did not mention the default `test` folder.

Reply:
@benjamincburns it should fail fast and quick in this case

Reply:
test